test/smoke_test/bcda_client.go

```diff
@@ -78,14 +79,18 @@ func getAccessToken() string {
 	return t.AccessToken
 }
 
-func startJob(resourceType string) *http.Response {
+func startJob(endpoint, resourceType string) *http.Response {
 	client := &http.Client{}
 	var url string
 
+	if endpoint != "Patient" {
```

Comment:
I think it might be better to take `endpoint` as it is passed in.  Allows for the test client to be more flexible... i.e., add more endpoints, test error cases (passing in bad endpoints), etc.
Suggest doing something like:
```
if endpoint == "Group" {
     groupId = "all"
}
```

Reply:
yeah lol this is better. will do 👍